src/main/java/dan200/computercraft/shared/common/BlockGeneric.java

```diff
@@ -219,12 +219,6 @@ public final AxisAlignedBB getBoundingBox( IBlockState state, IBlockAccess world
     }
 
     @Override
-    public final AxisAlignedBB getSelectedBoundingBox( IBlockState state, World world, BlockPos pos )
```

Comment:
I'm happy to, though that is the default behaviour already. I guess it might be a tiny bit quicker as it doesn't have to go via `IBlockState` again.